src/env.js

```diff
@@ -2,6 +2,6 @@
   window.__env = window.__env || {};
   // these placeholders are actually used to find and replace to set environment variables.
   // DO NOT CHANGE!!
-  window.__env.apiUrl = 'api';
+  window.__env.apiUrl = '/api';
```

Comment:
This is the API of the backend, right? We assume this is always at `/`? 🤔

Reply:
Dat wordt gebruikt voor lokaal development, het wordt overschreven binnen entrypoint.sh wanneer het onder docker gedraait wordt.

Reply:
Yes, this is our back-end. The default assumption now is `/api`, so that `demo.sensorenregister.nl/viewer/` wil look at `demo.sensorenregister.nl/api/` instead of `demo.sensorenregister.nl/viewer/api/`. This can still be overridden by setting the appropriate ENV variable.